upgrade/steps-helpers.go

```diff
@@ -351,3 +352,113 @@ func getRepoExpectedLocation(ctx Context, cwd, repoPath string) (string, error)
 
 	return filepath.Join(ctx.GoPath, "src", expectedLocation), nil
 }
+
+// Fetch the expected upgrade target from github. Return a list of open upgrade issues,
+// sorted by semantic version. The list may be empty.
+//
+// The second argument represents a message to describe the result. It may be empty.
+func GetExpectedTarget(ctx Context, name, upstreamOrg string) (*UpstreamUpgradeTarget, string, error) {
+	if ctx.TargetVersion != nil {
+		return &UpstreamUpgradeTarget{Version: ctx.TargetVersion}, "", nil
+	}
+	if !ctx.InferVersion {
+		return getExpectedTargetLatest(ctx, name, upstreamOrg)
+	}
+	return getExpectedTargetFromIssues(ctx, name)
+}
+
+func getExpectedTargetLatest(ctx Context, name, upstreamOrg string) (*UpstreamUpgradeTarget, string, error) {
+	latest := exec.CommandContext(ctx, "gh", "release", "list",
+		"--repo="+upstreamOrg+"/"+ctx.UpstreamProviderName,
+		"-L=1",
```

Comment:
Long arguments are clearer.
```suggestion
		"--limit=1",
```